pkg/new-ui/v1beta1/frontend/src/app/pages/experiment-creation/hyper-parameters/hyper-parameters.component.ts

```diff
@@ -0,0 +1,42 @@
+import { Component, Input } from '@angular/core';
+import { FormArray } from '@angular/forms';
+
+@Component({
+  selector: 'app-form-hyper-parameters',
+  templateUrl: './hyper-parameters.component.html',
+  styleUrls: ['./hyper-parameters.component.scss'],
+})
+export class FormHyperParametersComponent {
+  @Input() hyperParamsArray: FormArray;
+
+  constructor() {}
+
+  get combinations() {
```

Comment:
I think we should count count of current combinations only if search space is discrete.

Reply:
Is your concern about cases where the total number of Trials might be larger, since the Suggestions Controller might handle parameters differently? For example as you mentioned before, if the `step` gets omitted then the Suggestion Controller might have its own logic for selecting the step each time.

If that's the case, do you think it would make sense to change a little bit the phrasing in the UI to something like:
```
There are **approximately** 576 different combinations from these Hyper Parameters.
```
And maybe also have an annotations that explain that in some cases the number might be higher.

This way we at least provide some rough estimate of the number of Trials that will be created.

Reply:
I am not sure how we can calculate number of combinations if step is omitted.
For example, in skopt we use log-uniform distribution: https://github.com/kubeflow/katib/blob/master/pkg/suggestion/v1beta1/skopt/base_service.py#L44 where we don't know how many combinations algorithm can generate.
Is that correct @gaocegege ?

Reply:
ACK! Would it make sense to then only calculate the combinations if the `step` is present in all [ int, double ] variables?

Trying to think of how we could at least show this approximation in some cases.

Reply:
Sure, in that case we can calculate.

Reply:
I made this text conditional to appear only if no `step` field is omitted. 

![Screenshot from 2021-02-26 01-11-01](https://user-images.githubusercontent.com/11134742/109231585-8ea44d00-77cf-11eb-8576-c37db0331b9b.png)
![Screenshot from 2021-02-26 01-06-15](https://user-images.githubusercontent.com/11134742/109231593-919f3d80-77cf-11eb-8485-018b5a92b0f2.png)

Reply:
Looks great, thank you @kimwnasptd !